op-batcher/batcher/driver.go

```diff
@@ -313,6 +319,67 @@ func (l *BatchSubmitter) calculateL2BlockRangeToStore(ctx context.Context) (eth.
 // Submitted batch, but it is not valid
 // Missed L2 block somehow.
 
+func (l *BatchSubmitter) checkLeaderElectionBatcherStatus() (bool, error) {
```

Comment:
Call this something like `...IsLeader` to make it clearer what true/false means?